### DISCUSSION TITLE
how to use local lowdefy cli?

### DISCUSSION THREAD

**Discussion by wantingjun** [lowdefy/lowdefy#716] [Questions and Help] [answered]
hello,

I have some questions about how to install and use local lowdefy cli.
 
I want to locally install lowdfy cli, as in the tutorial, I run 'npm install lowdefy'. And then, what command can be  used to use **local cli** to initialize an app rather than using `npx` to get the latest version? 

For example, I try the  `lowdefy init`, but it didn't work, and the error in termianl is as below.
```
bash: lowdefy: command not found

```

Thank you!!

**Comment by Gervwyk** [comment, marked as answer]
Hi @wantingjun 

For me, `lowdefy init` works. However, you first need to globally install the Lowdefy package:
```
npm lnstall lowdefy -g
```
Then you can check your lowdefy version by running `lowdefy -v` which should return `3.19.0`. Note that if you globally install the lowdefy package you'd need to update the package for new version releases of lowdefy. This can be done by running:
```
npm update lowdefy -g
```
If you get a version response from `lowdefy -v` you should be able to run `lowdefy init`.

> Note the running `npx lowdefy@latest init` will always run the latest version of the Lowdefy CLI.

**Reply by SamTolmay** [reply]
As an extra note, running `npm install lowdefy -g` will install the CLI globally, so it will be available anywhere on tour PC. If you want to install it for a specific project (Maybe you want to install an older version for a specific project, for example), you can run `npm install lowdefy`. But then you need to have a [`package.json`](https://docs.npmjs.com/cli/v7/configuring-npm/package-json) file in your project directory.
